app/views/content/shared/quotes/_quote_progression_gloria.html.erb

```diff
@@ -0,0 +1,5 @@
+<%= render Content::QuoteComponent.new(
+  text: "I started out as a regular classroom teacher, but over the years, I’ve grown and taken on different roles. I became head of year to understand how to support students beyond academics, and later, I moved into a leadership role in teaching maths. Every step of the way, I’ve learned so much about how students learn, and I’m still learning every day.",
+  name: "Gloria, maths lead practitioner",
+  large: false
```

Comment:
False is the default for the quote component, so you should be able to remove this line